### PR TITLE
Adjust line-length of transport related classes to coding standard

### PR DESCRIPTION
**Pull request by danielmitterdorfer**

### CODE REVIEW

**Comment by danielmitterdorfer** [comment]
I have shortened the line-length of a few transport related classes. While I could have done that in #17133, the original PR was already big enough so I did this separately.

File: core/src/main/java/org/elasticsearch/http/netty/NettyHttpServerTransport.java [hunk 1]
```diff
@@ -265,8 +266,10 @@ public NettyHttpServerTransport(Settings settings, NetworkService networkService
         }
         this.maxContentLength = maxContentLength;
 
-        logger.debug("using max_chunk_size[{}], max_header_size[{}], max_initial_line_length[{}], max_content_length[{}], receive_predictor[{}->{}], pipelining[{}], pipelining_max_events[{}]",
-                maxChunkSize, maxHeaderSize, maxInitialLineLength, this.maxContentLength, receivePredictorMin, receivePredictorMax, pipelining, pipeliningMaxEvents);
+        logger.debug("using max_chunk_size[{}], max_header_size[{}], max_initial_line_length[{}], max_content_length[{}], " +
+            "receive_predictor[{}->{}], pipelining[{}], pipelining_max_events[{}]",
+                maxChunkSize, maxHeaderSize, maxInitialLineLength, this.maxContentLength,
```
Comment (jpountz) [inline core/src/main/java/org/elasticsearch/http/netty/NettyHttpServerTransport.java]: mhy isn't it aligned with the line above?

**Comment by jpountz** [comment]
I found one minor oddity. Otherwise LGTM.

File: core/src/main/java/org/elasticsearch/http/netty/NettyHttpServerTransport.java (hunk 1 above)
Comment (danielmitterdorfer) [inline core/src/main/java/org/elasticsearch/http/netty/NettyHttpServerTransport.java]: I guess IDEA did this. I'll check again.

**Comment by danielmitterdorfer** [comment]
@jpountz Thanks for the review! I'll check and merge afterwards.

**Pull request merged**